docs/content/toolbar.md

```diff
@@ -0,0 +1,68 @@
+# The SpatialOS Unreal GDK toolbar
+
+The SpatialOS Unreal GDK toolbar is an easy way to run and alter [`spatial` command-line tool](https://docs.improbable.io/reference/13.1/shared/glossary#the-spatial-command-line-tool) (SpatialOS documentation) commands from inside the Unreal Editor.
```

Comment:
Ah, so it would be:

> SpatialOS Unreal GDK toolbar is an easy way...

?

That reads very strangely to me... or did you mean the first word should be something else?

Reply:
I think The should be there. it reads weird without it

Reply:
@anne-edwards I meant the heading on line 1. I agree that on line 3 there should be a `The`.

Reply:
Ah, OK - I guess we don't say "The Interop Code Generator". I'll change this 👍